bitbots_docs/docs/manual/hardware/electronics.rst

```diff
@@ -0,0 +1,50 @@
+==========
+Electronics
+==========
+
+Powerboard
+==========
+
+The Powerboard allows to connect either an external power supply or a battery. It has a switch to turn on and off the power to the servos.
+
+* Schematic :download:`pdf <elektronik/wolfgang_power.pdf>`
+* PCB combined :download:`pdf <elektronik/wolfgang_power_pcb.pdf>`
+* PCB Top :download:`pdf <elektronik/wolfgang_power_pcb_top.pdf>`
+* PCB Bottom :download:`pdf <elektronik/wolfgang_power_pcb_bottom.pdf>`
+* PCB Silk :download:`pdf <elektronik/wolfgang_power_pcb_silk.pdf>`
+
+The footprint of the mosfet is currently wrong, but since the Wolfgang Core board integrates the functionality of this board, it will not be changed.
+
+
+DXLBoard
+========
+
+The DXLBoard is developed by Rhoban from Bordeaux. It communicates with the Servo bus over RS485 or TTL and the NUC via USB.
+
+Documentation about the Hardware is available here_. 
```

Comment:
Does this link work?
Github renders it as underlined text, but it is not clickable.

Reply:
This style of link worked in the previous version of the document,